boxsdk/session/session.py

```diff
@@ -242,8 +249,11 @@ def with_default_network_request_kwargs(self, extra_network_parameters):
         kwargs['default_network_request_kwargs'].update(extra_network_parameters)
         return self.__class__(**kwargs)
 
+    # pylint: disable=unused-argument
```

Comment:
Could you add a comment here to explain why this is being done?  We wouldn't want someone else to see this and think the unused argument can just be removed